test/fixtures/es-class-assign-property/expected-remove-context-wrap-es6.js

```diff
@@ -0,0 +1,23 @@
+class Foo1 extends React.Component {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+process.env.NODE_ENV !== "production" ? Foo1.childContextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+class Foo2 extends React.PureComponent {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
```

Comment:
I don't get it, this is a breaking change. How is this related to #77? Sounds like a different use case. I'm not sure we want that output.